src/containers/Application/ApplicationPage.tsx

```diff
@@ -54,36 +60,35 @@ const ApplicationPage: React.FC<ApplicationProps> = ({
 
   return (
     <>
-      <Header
-        as="h1"
-        textAlign="center"
-        content={currentUser?.organisation?.orgName || strings.TITLE_NO_ORGANISATION}
-      />
-      <Grid stackable style={inlineStyles.grid}>
-        <Grid.Column width={4}>
-          <ProgressBar
-            structure={fullStructure}
-            requestRevalidation={requestRevalidation as MethodRevalidate}
-            strictSectionPage={strictSectionPage as SectionAndPage}
-          />
-        </Grid.Column>
-        <Grid.Column width={10} stretched>
-          <Segment vertical style={{ marginBottom: 20 }}>
-            <Header content={fullStructure.sections[sectionCode].details.title} />
-            <PageElements
-              canEdit={current?.status === ApplicationStatus.Draft}
-              elements={getCurrentPageElements(fullStructure, sectionCode, pageNumber)}
-              responsesByCode={fullStructure.responsesByCode}
-              applicationData={fullStructure.info}
-              isStrictPage={
-                sectionCode === strictSectionPage?.sectionCode &&
-                pageNumber === strictSectionPage?.pageNumber
-              }
+      <Container id="application-form">
+        <Grid stackable className="dev-border">
```

Comment:
Why is this css called `dev-border`?

Reply:
Thanks, that was just for testing so I could see the div boundaries but I've removed the class now. FYI, I've put a more thorough "dev CSS" solution into #656 for doing this kind of thing going forwards.

Reply:
Nice! Will have a look after reviewing this one.